setup.py

```diff
@@ -36,12 +36,14 @@
         'monotonic>=1.6',
         'orderedset>=2.0.3',
         'phonenumbers~=8.12.12',
-        'pypdf2 ~= 3.0.1',
+        'pypdf >= 3.15.0',
         'python-json-logger~=2.0.7',
         'pytz>=2021.3',
         'pyyaml==5.4.1',
         'requests>=2.26.0',
         'smartypants>=2.0.1',
-        'statsd>=3.3.0',
+        'statsd>=3.3.0'
+        'statsd>=3.3.0'
+        'MarkupSafe>=2.1.3',
```

Comment:
as CodeQl shows, you added a line and so you are now missing a comma on line 46. Probably do not need one on line 47.